js/apps/admin-ui/public/locales/ca/common-help.json

```diff
@@ -1 +1,7 @@
-{}
\ No newline at end of file
+{
```

Comment:
This is somehow in the `ca` directory, I don't think that is intentional?

Reply:
@jonkoops I didn't edit this file. I am not sure what this file is. Can we remove it?